apps/site/src/app/(home)/sections/FAQ/FAQ.tsx

```diff
@@ -1,12 +1,14 @@
+import { Container } from "react-bootstrap";
```

Comment:
Chore: I forget if Next.js handles this well, but to be safe, we'll want to import just the individual component, i.e. `import Container from "react-bootstrap/Container"`
> Import individual components rather than the entire library. Doing so pulls in only the specific components that you use ([React Bootstrap, Importing Components](https://react-bootstrap.github.io/docs/getting-started/introduction#importing-components))